docs/guides/access-control.mdx

```diff
@@ -0,0 +1,256 @@
+
+For more information on setting up a local ReductStore instance, see the **[Getting Started](/docs/getting-started.mdx)** guide.
+
+
+### Creating a Token
+
+An access token can be created using the SDKs, CLI client, Web Console, or REST API. The token name must be unique within the store, and a client must have full access permission. You can also provision a token using environment variables. See the examples below:
+
+
+import CreatTokenCLI from "!!raw-loader!./examples/cli/access_control_create_token.sh";
+import CreateTokenPy from "!!raw-loader!./examples/py/src/access_control_create_token.py";
+import CreateTokenJs from "!!raw-loader!./examples/js/src/access_control_create_token.mjs";
+import CreateTokenRust from "!!raw-loader!./examples/rs/examples/access_control_create_token.rs";
+import CreateTokenCpp from "!!raw-loader!./examples/cpp/src/access_control_create_token.cc";
+import CreateTokenCurl from "!!raw-loader!./examples/curl/access_control_create_token.sh";
+import CreateTokenProvisioned from "!!raw-loader!./examples/provisioning/access_control_create.yml";
+
+<Tabs>
+    <TabItem value="CLI">
+        <CodeBlock language="bash">
+            {CreatTokenCLI}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="Web Console">
+        Steps to create a new access token using the Web Console:
+        1. Open the Web Console at **[http://127.0.0.1:8383](http://127.0.0.1:8383)** in your browser.
+        2. Click on the **"Security"** tab in the left sidebar.
+        3. Click on the plus icon in the top right corner to create a new token:
+
+        ![Create Access Token](./img/access_control_create_token.webp)
+
+        4. Fill in the token name and select the permissions for the token:
+
+        ![Create Access Token Form](./img/access_control_create_token_settings.webp)
+
+        5. Click on the **"Create"** button to create the token.
+
+        6. Copy the token value and save it in a secure place.
+
+        ![Access Token Created](./img/access_control_create_token_result.webp)
+    </TabItem>
+    <TabItem value="Python">
+        <CodeBlock language="python">
+            {CreateTokenPy}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="JavaScript">
+        <CodeBlock language="javascript">
+            {CreateTokenJs}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="Rust">
+        <CodeBlock language="rust">
+            {CreateTokenRust}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="C++">
+        <CodeBlock language="cpp">
+            {CreateTokenCpp}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="cURL">
+        <CodeBlock language="bash">
+            {CreateTokenCurl}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="Provisioning">
+        <CodeBlock language="yaml">
+            {CreateTokenProvisioned}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+
+:::info
```

Comment:
can be a warning

Reply:
I keep it consistent with the other guides. Warning is for operations that can remove data.